extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -1010,40 +1011,73 @@ export class AmpAutocomplete extends AMP.BaseElement {
   /**
    * Writes the selected value into the input field.
    * @param {?Element} element
+   * @return {?string}
    * @private
    */
-  selectItem_(element) {
+  setInputValue_(element) {
     if (element === null || element.hasAttribute('data-disabled')) {
-      return;
+      return null;
     }
     const selectedValue =
       element.getAttribute('data-value') || element.textContent;
```

Comment:
It looks like `textContent` can return `null`, let's add `.. || ''` to the end of this line for type safety on `selectedValue`.